tsconfig.json

```diff
@@ -1,27 +1,15 @@
 {
```

Comment:
Just the bare minimum here. You'll note that I'm targeting ES2020, with module set to ES 2022. This is to match D1 lib, which supports errors with a cause, which is an ES 2022 thing in typescript. Additionally, we are supporting DOM APIs and ES 2022 features. Note that I'm only including source here in the build. The fun thing about mocha is that it will auto-compile our tests even if they aren't included here (though we do include tests in our lint tsconfig).